playground/mocks/data/idp/idx/error-poll-400-device-account-invalid.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version":"1.0.0",
+  "stateHandle":"eyJ6aXAiOiJERUYiLCJhb",
+  "expiresAt":"2021-11-18T20:16:13.000Z",
+  "intent":"LOGIN",
+  "messages":{
+    "type":"array",
+    "value":[
+      {
+        "message":"Your device or account was invalidated. If this is unexpected, contact your administrator for help.",
+        "i18n": {
+          "key": "core.auth.factor.signedNonce.error.deletedDevice"
```

Comment:
This key isn't listed in our `login.properties` or being transformed in [i18nTransformer.js](https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/i18nTransformer.js), so the error message will likely default to the `message` value here, resulting in English leaks.

I may be wrong, but it looks like you're trying to surface an error message different from the server. I'd suggest using the [i18nTransformer.js](https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/i18nTransformer.js)'s `I18N_OVERRIDE_MAPPINGS` object to handle this use case. When the Widget sees an error with the key `core.auth.factor.signedNonce.error.deletedDevice`, it can remap it to the expected `user.fail.verifyIdentity` message.

Reply:
Updated